### PR TITLE
Added startup

### PR DESCRIPTION
**Pull request by Neeraj-2307**

### CODE REVIEW

File: data/surveysparrow.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: SurveySparrow
+description: SurveySparrow is an end-to-end conversational experience management platform that helps in getting a 40% better response rate.
+founded: October 1 2017
+based: Both
+domain: Computer And Related Activities
+logo: surveysparrow.jpeg
```
Comment (lemokami) [inline data/surveysparrow.mdx:7]: ❤️ Thanks for your conribution but there is a tiny problem with the filename.  It's supposed to be `SurveySparrow.png` right?

Comment (Neeraj-2307) [reply data/surveysparrow.mdx]: Yes, fixed it now 😃

Comment (lemokami) [reply data/surveysparrow.mdx]: Thank you✨, I appreciate it.
youre-welcome-pleasure

**Pull request merged**